ci-operator/step-registry/openshift-extended/test/disruptive/openshift-extended-test-disruptive-ref.yaml

```diff
@@ -10,7 +10,7 @@ ref:
     default: |
         MCO|ETCD|PSAP|SDN|STORAGE|API_Server|Authentication|Cluster_Operator
         Cluster_Infrastructure|OLM|Network_Edge|Operator_SDK
-        Workloads|Image_Registry|Container_Engine_Tools
```

Comment:
@sgoveas the Jira component name is `Installer` not `INSTALLER`.   So I think you need to change that to match what's represented in JIRA, otherwise problems will occur for scripts that key off the name `Installer`

Reply:
it is used to selected case. Installer is too common, so that other cases will be treated as installer case.
so, we use INSTALLER.
actually it is not only case. for example, Storage is jira component, but here we use STORAGE.